packages/expo-video/src/VideoPlayer.types.ts

```diff
@@ -40,6 +40,28 @@ export declare class VideoPlayer extends SharedObject<VideoPlayerEvents> {
    */
   currentTime: number;
 
+  /**
+   * The current time of the video as a date. The system calculates this value from the `EXT-X-PROGRAM-DATE-TIME` tag.
+   * > This property is get-only
+   * @platform ios
+   * @platform android
+   */
+  readonly currentDate: number;
+
+  /**
+   * Float value indicating the latency of the live stream in seconds.
+   * > This property is get-only
+   * @platform ios
+   * @platform android
```

Comment:
```suggestion
   * @platform android
   * @platform ios
```

Same as previous comment.